### ISSUE TITLE
Elektra Release before or after survey

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#93]
Do you think it will make a difference if the release of Elektra 0.8.17 is before or after the survey release?

If doing it together news items might then have a focus on the release and people interested in the survey but not into Elektra might not see it. On the other hand some news (e.g. pro-linux) might only find the release and not the survey worth of reporting?

What do you think?

**Comment by Namoshek** [comment]
I don't think this is really relevant. We can still give a hint to the survey with the next release notes of Elektra.

**Comment by sirblackheart** [comment]
I think we should release 0.8.17 before or with the survey. Maybe even give a road map what is going to come for 0.8.18?

**Comment by markus2330** [comment]
@sirblackheart when releasing before the link would not be in the release notes.

@Namoshek The next release will be when the complete survey is over.

**Comment by omnidan** [comment]
I think it would be good to release with the survey and include a link in the release notes, especially because our target audience will read these (developers that are interested in configuration systems)
